scripts/contrithanks.sh

```diff
@@ -28,8 +28,13 @@
 
 start=$1
 
-if test -z "$start"; then
+if test "$start" = "-h"; then
   echo "Usage: $0 <since this tag/hash>"
+  exit
+fi
+if test -z "$start"; then
+  start=`git tag --sort=taggerdate | tail -1`;
+  echo "Since $start:"
```

Comment:
I don't think this script should add `Since...` because the output of this script is basically an updated version of `docs/THANKS` and that line ruins that convenience (and will probably just lead to me missing to remove that line when I update THANKS in the future)

Reply:
That makes sense. I've removed that line now.